jax/interpreters/ad.py

```diff
@@ -162,7 +167,9 @@ def recast_to_float0(primal, tangent):
     return tangent
 
 # NOTE: The FIXMEs below are caused by primal/tangent mixups (type errors if you will)
-def backward_pass(jaxpr: core.Jaxpr, reduce_axes, consts, primals_in, cotangents_in):
+def backward_pass(jaxpr: core.Jaxpr, reduce_axes, transform_stack, consts, primals_in, cotangents_in):
```

Comment:
Here and elsewhere, it's unfortunate to reuse the name `transform_stack` for a bool rather than an actual transform stack. How about another name, like `extend_transform_stack` or something?